### PR TITLE
Added protoc-gen-validate in test Vagrantfile

### PR DESCRIPTION
**Pull request by eloycoto**
- CleanWorkspace in Jenkins Nightly

### CODE REVIEW

**Comment by eloycoto** [comment]
test-nightly

File: test/provision/runtime_install.sh [hunk 1]
```diff
@@ -8,5 +8,12 @@ DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 source "${PROVISIONSRC}/helpers.bash"
 
+export GOPATH="/home/vagrant/go"
+sudo -E /usr/local/go/bin/go get -d github.com/lyft/protoc-gen-validate
+
+cd ${GOPATH}/src/github.com/lyft/protoc-gen-validate
+sudo git checkout 930a67cf7ba41b9d9436ad7a1be70a5d5ff6e1fc
```
Comment (rlenglet) [inline test/provision/runtime_install.sh:17]: Could you please add comments to remind keeping this file in sync with the top-level Vagrantfile, and vice-versa?

**Review by rlenglet — APPROVED** [review]

**Comment by rlenglet** [comment]
And thanks for fixing this!

**Review by aanm — APPROVED** [review]
Can you create a PR in https://github.com/cilium/packer-ci-build with these changes?

File: test/provision/runtime_install.sh (hunk 1 above)
Comment (nebril) [reply test/provision/runtime_install.sh:17]: I would go a step further and put this hash in a variable in sourced file that is used by both this and to-level Vagrantfile.

**Review by nebril — CHANGES_REQUESTED** [review]

**Review by nebril — APPROVED** [review]
On second thought, this seems to be blocking, let's address the hash issue in another PR.

**Comment by eloycoto** [comment]
Hi, 

PR in the base image done, I'll update in the next round, added the comment in the code too. 
https://github.com/cilium/packer-ci-build/pull/33

This PR is blocking the green light in the Nightly test, so it'll be awesome to get it merged asap ;-)

**Comment by eloycoto** [comment]
test-me-please

**Pull request merged**